modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -199,6 +228,14 @@ class Stats_Behavioural extends \NDB_Form
             $vl     = $row['VLabel'];
             $c      =& $this->tpl_data['dde']['C' . $center];
 
+            if (is_null($c)) {
```

Comment:
remove 231-238
when you get a null value inside this loop, it will throw new Exception on the page, then broke the page.